### PR TITLE
README.md link update

### PR DESCRIPTION
**Pull request by Anthony-B-Russo10**
Updated the links to include Levin's mobile Celestia archives and the recently created Cesletia.mobi site for mobile addons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +25,7 @@ either version 2 of the License, or (at your option) any later version.
 This program is distributed in the hope that it will be useful, but WITHOUT
 ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details,
-which you should have received along with this program (filename: COPYING).
+which you should have received along with this program (filename: elestiaCOPYING).
```
Comment (375gnu) [inline README.md:28]: please remove garbage

**Review by 375gnu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Anthony-B-Russo10) [reply README.md]: File name reset back to the correct file name found in the repository.

**Review by 375gnu — CHANGES_REQUESTED** [review]
please merge commits

**Review by 375gnu — CHANGES_REQUESTED** [review]
why did you made 3rd commit?

**Review by 375gnu — CHANGES_REQUESTED** [review]
Once again: please squash all commits into a single one.

**Comment by Anthony-B-Russo10** [comment]
I have looked, I have not found any way to squash the commits from the website.

**Comment by 375gnu** [comment]
don't use website. use git or github client

**Comment by Anthony-B-Russo10** [comment]
Tried the Github client, still didn't work.

**Comment by 375gnu** [comment]
closing as no activities

**Pull request closed without merging**